msg/scratch_msgs.js

```diff
@@ -17567,10 +17682,10 @@ Blockly.ScratchMsgs.locales["zh-cn"] =
     "MV2_WALK": "%1 %2 走 %3 步，步距 %4 毫米, 时间 %5 秒，转向 %6 度",
     "MV2_TURN": "%1 %2 向 %4 转 %3 步",
     "MV2_WIGGLE": "%1 %2 跳摆摆舞",
-    "MV2_CIRCLE": "%1 %2 %4 秒内向 %3 跳圈圈舞",
+    "MV2_CIRCLE": "%1 %2 %3 秒内向 %4 跳圈圈舞",
```

Comment:
I notice the order of the arguments  has changed from 1,2,4,3 to 1,2,3,4 for this and zh-tw.
Just checking that was on purpose - was it wrong before?